src/handlers/registration.py

```
@@ -223,11 +223,13 @@ async def step_mail(m: Message, state: FSMContext):
                 record_message(m.from_user.id, msg, "command")
         
         except pymysql.err.IntegrityError as e:
-            logger.error(f"Не удалось зарегистрировать пользователя с e-mail {email}: {e}")
+            logger.error(f"РќРµ СѓРґР°Р»РѕСЃСЊ Р·Р°СЂРµРіРёСЃС‚СЂРёСЂРѕРІР°С‚СЊ РїРѕР»СЊР·РѕРІР°С‚РµР»СЏ СЃ e-mail {email}: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)